### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by peregrines2** [Louisvdw/dbus-serialbattery#432] [Q&A] [answered]
How can dbus-serialbattery be uninstalled?
I want to completely reinstall it and therefore first remove it.
Can I just delete the folder in /etc?
Or is there a trick to do a clean uninstall/reinstall?

THANK YOU

**Comment by Louisvdw** [comment, marked as answer]
Hi
If you just want to reinstall it you can just delete the folder and install again.
You could also [disable](https://github.com/Louisvdw/dbus-serialbattery/wiki/How-to-install#disable-the-driver) it first, but worry about that.

**Comment by peregrines2** [comment]
Thank You!!! :)
